src/Components/Footer/Footer.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import Classes from './Footer.module.css';
+
+// icons import
+import {IoMail} from 'react-icons/io5';
+import {BsTelephoneFill} from 'react-icons/bs';
+import {MdLocationOn} from 'react-icons/md';
+import {BsFacebook} from 'react-icons/bs'
+import {FaLinkedinIn} from 'react-icons/fa'
+import {GrInstagram} from 'react-icons/gr';
+import {IoMdSend} from 'react-icons/io';
+
+
+
+function Footer() {
+    return (
+        <div className={Classes.container}>
+             <div className={Classes.quickLinks}>
+                 <div className={Classes.heading}>
+                     Quick Links
+                 </div>
+                 <ul className={Classes.list1}>
+                    <li className={Classes.listItem}>Blah</li>
+                    <li className={Classes.listItem}>Blah</li>
+                    <li className={Classes.listItem}>Blah</li>
+                    <li className={Classes.listItem}>Blah</li>
+                    <li className={Classes.listItem}>Blah</li>
+                 </ul>
+             </div>
+             <div className={Classes.contactUs}>
+             <div className={Classes.headingContact}>
+                     Contact Us
+                 </div>
+                 <ul className={Classes.list}>
+                    <li className={Classes.listItem2}> <IoMail color= {'white'} /><span  className={Classes.text}>mail-id@domain.com</span> </li>
```

Comment:
change this to an `<a href="mailto:contact@interiit-tech.org">contact@interiit-tech.org</a>`